lib/src/scss-shared/_nx-colors.scss

```diff
@@ -64,14 +64,15 @@ $nx-fail-color-hover: #5d0000;
 */
 
 // Structural
-$nx-site-background-color: $nx-grey-1;
+$nx-site-background-color: #f4f5f9;
 $nx-page-background-color: $nx-grey-1;
 $nx-sidebar-background-color: #f1f3fa;
 $nx-content-background-color: $nx-grey-1;
 $nx-header-border: #f1f3fa;
 
 //  Text
-$nx-text-color: $nx-off-black;
+$nx-heading-color: #2C2C2C;
+$nx-text-color: #474747;
```

Comment:
There's a usage of this color in `_nx-page-header.scss` that should now be replaced by the variable

Reply:
> There's a usage of this color in `_nx-page-header.scss` that should now be replaced by the variable

https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/9e4686a519ddd1505191432663684b7eb11de060